install.sh

```diff
@@ -5,11 +5,14 @@
 export ALIGN_HOME=$PWD
 export ALIGN_WORK_DIR=$ALIGN_HOME/work
 
+export SUDO=sudo
```

Comment:
Is it okay if we check whether the user is root or not? This could enable us to start a fresh ubuntu container and run install w/o any modifications.